### DISCUSSION TITLE
How to rearrange order of images in image property

### DISCUSSION THREAD

**Discussion by Bluepuper** [pocketbase/pocketbase#1846] [Q&A] [answered]
I have property `images`, and when i chose images (did it by holding `cmd` and clicking in right order) it uploaded in random order, how can i rearrange that order?

**Comment by arslanastral** [comment, marked as answer]
What pocketbase version are you using? I've tested it locally on the `0.12.2` and it keeps the order in which you select images. You can also try prefixing files with a number like `file1.png`,`file2.png` etc but it should be working without this.

Also, there isn't a way yet to manually rearrange the order though the admin UI. Most of the time you will be using the sdk's sort or your own sort at application level.

**Reply by Bluepuper** [reply]
i'm using 0.9.2. Tried to upload some images and pocketbase auto ordered them in asc alphabetic order. But i then i checked previously created rows and there is desc order somehow
![Screen Shot 2023-02-13 at 10 27 06 AM](https://user-images.githubusercontent.com/79052139/218396355-0790c36c-7320-444a-9d95-7191070dcef0.png)
well, i will sort it at application level, thanks for some info
